language-server/modules/langserver-core/src/test/resources/completion/class_def/config/config25.json

```diff
@@ -493,24 +493,6 @@
       "sortText": "R",
       "insertText": "byte",
       "insertTextFormat": "Snippet"
-    },
-    {
-      "label": "new(int field1, int field2)",
-      "kind": "Function",
-      "detail": "()",
-      "documentation": {
-        "right": {
-          "kind": "markdown",
-          "value": "**Package:** _ballerina/module1:0.1.0_  \n  \n  \n**Params**  \n- `int` field1  \n- `int` field2"
```

Comment:
This language server's completions have been failed. We need to check how this compiler changes have been affected

Reply:
@nadeeshaan isn't this due to the changes made in the filtering mechanism at `visibleSymbols()` method of the Semantic API?

Reply:
Updated with a fix.